.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
-  - "8.11.4"
+  - "10"
```

Comment:
Actually, can you set the node version to 10.13.3 which is the current LTS version and add 12.12.0 (current)?
Also, you don't need the quotation marks 😄 

```
node_js:
  - 10.13.3
  - 12.12.0
```